diff --git a/helfi_formtool_embed.module b/helfi_formtool_embed.module
index 68580c8..b77dbfe 100644
--- a/helfi_formtool_embed.module
+++ b/helfi_formtool_embed.module
@@ -71,11 +71,8 @@ function helfi_formtool_embed_theme() {
  */
 function helfi_formtool_embed_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
 
-
+  /*
   if ($form_id == 'user_login_form') {
-    // @todo here we need to clean up form in select places. don't know how though.
-
-    /** @var \Drupal\node\Entity\Node $node */
     $node = \Drupal::routeMatch()->getParameter('node');
 
     if ($node !== NULL && $node->getType() == 'form_embed_page') {
@@ -85,7 +82,7 @@ function helfi_formtool_embed_form_alter(&$form, \Drupal\Core\Form\FormStateInte
       $d = 'asdf';
     }
   }
-
+  */
 }
 
 /**
